mailpoet/assets/js/src/automation/editor/store/actions.ts

```diff
@@ -121,6 +122,21 @@ export function* deactivate() {
       },
     );
   }
+  if (
+    !deactivateWorkflowRuns &&
+    data?.data.status === WorkflowStatus.DEACTIVATING
+  ) {
+    void createNotice(
+      'success',
+      __(
+        'Automation is deactivated. But recent users are still going through the flow.',
+        'mailpoet',
+      ),
```

Comment:
Did we have any string in specs? I'd probably remove the `But` at the beginning of the second sentence. Or we can let @NeosinneR chime in here.

Reply:
Was taken from Figma: 
![image](https://user-images.githubusercontent.com/6458412/197721905-3c553d21-d92d-44a4-b3f4-b7a98679bd53.png)

Reply:
Feels a bit off (I'd also either remove `But` or keep it as one sentence), but I'm also okay keeping it. @websupporter your choice.